server/utils/comments/comments.util.js

```diff
@@ -16,6 +17,29 @@ class commentValidation {
     }
     return next();
   }
+
+  /**
+   * Check User of article
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Object} next
+   * @returns {Object} response
+   */
+  static async checkUser(req, res, next) {
+    const { id } = req.params;
+    const userId = req.user.id;
+
+    const userComment = await Comment.findOne({
+      where: {
+        userId,
+        id
+      }
```

Comment:
Good job on this. However there is currently no check to see if the comment resides on the comments table, or the draftcomments table, so as to know the appropriate table to delete from. Please look into this

Reply:
It's been fixed. Thanks